FoodKeeper-API/build.gradle

```diff
@@ -15,8 +15,9 @@ apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
 
 bootJar {
-    baseName = 'gs-rest-service'
-    version =  '0.1.0'
+    baseName = 'foodkeeper-rest-service'
+    mainClassName = 'com.github.LibreFoodPantry.FoodKeeperAPI.App'
+    version =  '0.0.1'
```

Comment:
Why was 0.0.1 chosen as the version?

Is this not ready to be released as version 1 yet?

Reply:
Ahh.... I see it is from my comment to #5.

We should discuss versioning. How complete do you feel this is?

Reply:
I think it's mostly done, but maybe I should finish up the landing page before giving it a 1.0.0 version number. 

Also, I'm not certain if all the endpoints are correct. For example, the endpoint "/products" returns the entire list of products, which I'm not sure is necessary yet.

Reply:
Let's bump the version to 1.0.0. 

If I can change this directly on GitHub, I will, the merge it.

Reply:
Nope. I can't do it on GitHub.

@apham1 Please bump the version to 1.0.0, push the change, then I'll merge it.

Reply:
Fixed in commit 2fd8776